parm/config/gfs/config.com

```diff
@@ -74,6 +74,8 @@ declare -rx COM_WAVE_RESTART_TMPL=${COM_BASE}'/model/wave/restart'
 declare -rx COM_WAVE_PREP_TMPL=${COM_BASE}'/model/wave/prep'
 declare -rx COM_WAVE_HISTORY_TMPL=${COM_BASE}'/model/wave/history'
 declare -rx COM_WAVE_GRID_TMPL=${COM_BASE}'/products/wave/gridded'
+declare -rx COM_WAVE_GRIB_TMPL=${COM_BASE}'/products/wave/grib2'
```

Comment:
grib2 seems a bit odd to me since there's a "gridded"?   But if this is the structure that g-w wants, thats fine. Just seemed odd so I'm commenting.

Reply:
@KateFriedman-NOAA @aerorahul I need some guidance here in order to finalize where the ensemble wave products should end up. (see PR comments in how it was tested for more information). Your input is much appreciated. Thanks!